pkg/apis/migration/v1alpha1/directimagemigration_types.go

```diff
@@ -33,7 +33,7 @@ type DirectImageMigrationSpec struct {
 	DestMigClusterRef *kapi.ObjectReference `json:"destMigClusterRef,omitempty"`
 
 	// Holds names of all namespaces to run DIM to get all the imagestreams in these namespaces.
-	Namespaces        []string              `json:"namespaces,omitempty"`
```

Comment:
@ibolton336 seems like there may be an editor config issue here. We have a mix of vs code, goland, and probably vim with controller contributors. 

I can tell you that with VS code if you just installed go language support and use default settings it shouldn't be deleting all of this whitespace. I think this needs to be fixed otherwise we'll have thrashing in whitespace settings on every PR.

Reply:
@djwhatle This was actually happening when running 'make run' on my machine. I will take another look but I set up my vscode settings to turn all formatting off unless a config file was present for prettier.

Reply:
@djwhatle This actually happens when there's a comment on api fields. If there is no comment, all api fields are right aligned automatically by adding whitespaces, if there's a comment, the whitespaces are removed. I think go fmt is at fault here.

Reply:
Oh weird, thanks for pointing that out @pranavgaikwad , so it sounds like maybe after this change goes in we won't see things shift around anymore, just probably never ran `go fmt` after the field documentation that Shubham and Jaydip worked on.